charts/opensearch/Chart.yaml

```diff
@@ -15,7 +15,7 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 2.1.0
+version: 2.1.1
```

Comment:
Because the API (values file) is changing this is at least a minor version bump, rather than a bugfix.

Reply:
Yes this should be a minor version bump. I agree with @smlx